lib/llm/providers/anthropic.rb

```diff
@@ -44,5 +43,17 @@ def auth(req)
     def response_parser
       LLM::Anthropic::ResponseParser
     end
+
+    def transform_prompt(prompt)
+      return unless URI === prompt
```

Comment:
`nil` is not a valid prompt. Probably should be `return prompt unless URI === prompt`. Or:
```ruby
if URI === prompt
  # ...
else
  prompt
end
```